installation/language/de-DE/de-DE.ini

```
@@ -15,13 +15,13 @@ INSTL_STEP_SUMMARY_LABEL="Überblick"
 ;Language view
 INSTL_SELECT_LANGUAGE_TITLE="Sprachauswahl"
 INSTL_WARNJAVASCRIPT="Warnung! JavaScript muss für eine reibungslose Installation von Joomla! aktiviert sein!"
-INSTL_WARNJSON="Zur Joomla!-Installation ist ein aktives JSON in der PHP-Umgebung notwendig!"
+INSTL_WARNJSON="In der PHP-Umgebung muss JSON aktiviert sein, damit Joomla installiert werden kann!"
 
 ;Preinstall view
 INSTL_PRECHECK_TITLE="Installationsprüfung"
-INSTL_PRECHECK_DESC="Sollte nur eins der Einträge rechts vom Server nicht unterstützt werden, mit einem „<strong class=\"label label-important\">Nein</strong>“ gekennzeichnet, dann sollten die Einstellungen auf dem Server angepasst werden.<br />Joomla! kann nicht installiert werden, wenn die unten aufgeführten Systemvoraussetzungen nicht erfüllt sind."
+INSTL_PRECHECK_DESC="Sollte einer der Einträge vom Server nicht unterstützt werden, mit einem „<strong class=\"label label-important\">Nein</strong>“ gekennzeichnet, bitte die Einstellungen auf dem Server prüfen und anpassen.<br />Joomla! kann nicht installiert werden, wenn die unten aufgeführten Systemvoraussetzungen nicht erfüllt sind."
 INSTL_PRECHECK_RECOMMENDED_SETTINGS_TITLE="Empfohlene Einstellungen:"
-INSTL_PRECHECK_RECOMMENDED_SETTINGS_DESC="Diese Einstellungen werden für PHP empfohlen, um eine gute Kompatibilität mit Joomla! zu gewährleisten.<br />Jedoch kann Joomla! hier mit Einschränkungen in den Empfehlungen trotzdem funktionieren."
+INSTL_PRECHECK_RECOMMENDED_SETTINGS_DESC="Diese Einstellungen werden für PHP empfohlen, um eine gute Kompatibilität mit Joomla zu gewährleisten.<br />Jedoch kann Joomla! hier mit Einschränkungen in den Empfehlungen trotzdem funktionieren."
```

in dem string ist Joomla einmal mit und einmal ohne `!`?

```suggestion
INSTL_PRECHECK_RECOMMENDED_SETTINGS_DESC="Diese Einstellungen werden für PHP empfohlen, um eine gute Kompatibilität mit Joomla! zu gewährleisten.<br />Jedoch kann Joomla! hier mit Einschränkungen in den Empfehlungen trotzdem funktionieren."
```